src/pandablocks_ioc/_pvi.py

```diff
@@ -206,22 +209,19 @@ def create_pvi_records(record_prefix: str):
         # Create top level Device, with references to all child Devices
         device_refs = [DeviceRef(x, x) for x in pvi_records]
 
-        # # TODO: What should the label be?
         device = Device("TOP", device_refs)
         devices.append(device)
 
         # TODO: label widths need some tweaking - some are pretty long right now
+        # TODO: Need to decide how to handle already existing directory/files.
+        # Could still be left over stuff from a previous run?
         formatter = DLSFormatter(label_width=250)
-        with tempfile.TemporaryDirectory() as temp_dir:
-            for device in devices:
-                try:
-                    formatter.format(
-                        device,
-                        record_prefix + ":",
-                        Path(f"{temp_dir}/{device.label}.bob"),
-                    )
-                    with open(f"{temp_dir}/{device.label}.bob") as f:
-                        Pvi.bob_file_dict.update({f"{device.label}.bob": f.read()})
-
-                except NotImplementedError:
-                    logging.exception("Cannot create TABLES yet")
+        for device in devices:
+            try:
+                formatter.format(
+                    device,
+                    record_prefix + ":",
+                    Pvi._screens_dir / Path(f"{device.label}.bob"),
+                )
+            except NotImplementedError:
+                logging.exception("Cannot create TABLES yet")
```

Comment:
Is this error still valid in PVI 0.5? I'm not sure it's well worded anyway, as it could be anything throwing that error and we assume it was to do with tables...

Reply:
Yep, after pinning `pvi>=0.5` this isn't an issue. I'll remove the `try, except`